unified_planning/model/problem.py

```diff
@@ -362,12 +395,21 @@ def has_object(self, name: str) -> bool:
 
     def objects(self, typename: 'unified_planning.model.types.Type') -> List['unified_planning.model.object.Object']:
         '''Returns the objects of the given user types.'''
-        res = []
+        res:List['unified_planning.model.object.Object'] = []
```

Comment:
If we make this as an Iterator, shall we make every other method, like "fluents()", "actions()" etc. as an Iterator? @mikand

Reply:
Well, no because when you can simply return a list you already have stored in memory there is no point in forcing an iteration. Iterators make sense when you have to create a new list/set and return it